crates/aptos/src/common/init.rs

```diff
@@ -139,31 +139,37 @@ impl CliCommand<()> for InitTool {
         }
 
         // Fetch the top 5 (index 0-4) accounts from Ledger
-        let derivative_path = if self.ledger {
+        let derivation_path = if self.ledger {
             let account_map = aptos_ledger::fetch_batch_accounts(Some(0..5))?;
             eprintln!(
-                "Please choose a derivative path from the following {} ledger accounts, or choose an arbitrary path that you want to use:",
+                "Please choose an index from the following {} ledger accounts, or choose an arbitrary index that you want to use:",
                 account_map.len()
             );
-            for (index, account) in account_map.iter() {
-                eprintln!("Path: {} Address {}", index, account);
+
+            // Iterate through the accounts and print them out
+            for (index, (deri_path, account)) in account_map.iter().enumerate() {
+                eprintln!(
+                    "Index: {}, Derivatiion Path: {} Address: {}",
```

Comment:
My suggestion for the format. Also a typo fix.
```suggestion
                    "[{}] Derivation path: {} (address: {})",
```